CRM/Event/BAO/Participant.php

```diff
@@ -293,6 +293,15 @@ public static function create(&$params) {
     return $participant;
   }
 
+  public static function getAvailableSpaces(int $eventId, bool $includeWaitlist = TRUE): int {
```

Comment:
Make sure this function is marked as internal and not supported for external use - it seems the `BAO` functions might exist before the api is written but we don't want people to start using them as there is no intention that they would be supported for external use & need to be super clear in each comment block that people will need to wait for the api as these functions could change multiple times, or be abandoned but the api will be stable